keyboards/herringbone/config.h

```diff
@@ -0,0 +1,218 @@
+/*
+Copyright 2020 Ramon Imbao
+
+This program is free software: you can redistribute it and/or modify
+it under the terms of the GNU General Public License as published by
+the Free Software Foundation, either version 2 of the License, or
+(at your option) any later version.
+
+This program is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License
+along with this program.  If not, see <http://www.gnu.org/licenses/>.
+*/
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID 0xB16B
```

Comment:
[Another keyboard](https://github.com/qmk/qmk_firmware/tree/master/keyboards/wete) already uses this VID.  
That board also has VIA support, so another VID will be needed.

You can check the currently used VID/PID values either [here](https://yanfali.github.io/qmk_usb_usage/) or with `git grep`.

Reply:
This particular keyboard is mine too lol. Alright, I thought I could get away with having the same VID but different PID.

Reply:
Oh, that's a crucial info I did not know, sorry about that.  
In this case the same VID would make sense.  

If you are planning on creating and contributing other boards (which I hope you do), it would also make sense to create a "vendor" directory and place your boards under it.  
This is done the sooner, the better, because for already existing boards it's a breaking change.

tl;dr: You should use the same VID for all your stuff and maybe move this to `keyboards/ramonimbao/herringbone`.

You `wete` board should also be moved there, but as I said, it's going to be a breaking change.

Reply:
I'm only about to fulfill the boards I've made so far in the local GB I did, but I haven't shipped them out yet. Will this be not be considered a breaking change anymore in that case? The only ones that have the PCBs so far are me and another local guy.

Reply:
I think this can be an exception, but the others will weigh in on it.  
I recommend doing the move as soon as possible in this case.